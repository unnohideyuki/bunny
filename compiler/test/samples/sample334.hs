f x = case x of
  0 -> "zero"
  _ -> "non-zero"

main = do putStrLn $ f 1
          putStrLn $ f 0
          putStrLn $ f (1::Int)
          putStrLn $ f (0::Integer)
          putStrLn $ f (1.0::Double)
          putStrLn $ f (0.0::Float)

