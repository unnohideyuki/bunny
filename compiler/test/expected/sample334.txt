non-zero
zero
non-zero
zero
non-zero
zero
